massa-execution/src/vm.rs

```diff
@@ -194,10 +195,14 @@ impl VM {
             for (op_idx, operation) in block.operations.clone().into_iter().enumerate() {
                 let operation_sc = OperationSC::try_from(operation.content);
                 if operation_sc.is_err() {
-                    // only fail if the operation cannot parse the sender address
+                    // fails if the operation cannot parse the sender address or if the bytecode is invalid
                     continue;
                 }
                 let operation = &operation_sc.unwrap();
+
+                // Prepare context and save the Initial ledger changes before execution
```

Comment:
no need for the upper-case `Initial`, unless you're referring to some construct in the code...

Reply:
done